test/wast2json/module-binary.txt

```diff
@@ -0,0 +1,15 @@
+;;; TOOL: wast2json
```

Comment:
How does this test work?  I thought wast2json write a json file with binaries on the side.

Reply:
It does, but it also outputs the contents of the module if you pass `-v` too